src/Workspaces/Remote/ServiceHub/UserOperationBooster.cs

```diff
@@ -55,8 +63,8 @@ public void Dispose()
 
                 if (s_count == 0)
                 {
-                    // when boost is done, set process back to below normal priority
-                    Process.GetCurrentProcess().PriorityClass = ProcessPriorityClass.BelowNormal;
+                    // when boost is done, try setting process back to below normal priority
+                    Process.GetCurrentProcess().TrySetPriorityClass(ProcessPriorityClass.BelowNormal);
```

Comment:
Should the latch be used here too?

Reply:
I'd actually put the latch in `ProcessExtensions` so it's automatically used everywhere it's applicable

Reply:
Will do that change.